where_to_go/lib/classes/place_list_screen.dart

```diff
@@ -0,0 +1,75 @@
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+import "package:go_router/go_router.dart";
+import "../features/places/places_provider.dart";
+
+const beige = Color.fromARGB(255, 196, 194, 194);
+
+class PlaceListScreen extends ConsumerWidget {
+  const PlaceListScreen({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final places = ref.watch(placesProvider);
+
+    return Scaffold(
+      backgroundColor: beige,
+      appBar: AppBar(
+        title: const Text("Hiszpania - Barcelona", style: TextStyle(fontWeight: FontWeight.bold)),
+      ),
+      body: ListView.builder(
+        itemCount: places.length,
+        itemBuilder: (BuildContext context, int index) {
+          final place = places[index];
+
+          return InkWell(
+            onTap: () async {
+              await GoRouter.of(context).push("/dreamPlace/${place.id}");
```

Comment:
```suggestion
              await GoRouter.of(context).push("${DreamPlaceScreen.route}/${place.id}");
```
jak już masz taką zmienną to warto użyć żeby zredukować szansę na literówkę